pkg/set/set.go

```diff
@@ -0,0 +1,15 @@
+package set
+
+import (
+	"context"
+
+	"github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+)
+
+func SetProfile(ctx context.Context, clientsets *k8sutil.Clientsets, recoveryOption, operatorNamespace, storageClusterNamespace string) {
+
+	cephArgs := []string{"config", "set", "osd", "osd_mclock_profile", recoveryOption}
+
+	exec.RunCommandInOperatorPod(ctx, clientsets, "ceph", cephArgs, operatorNamespace, storageClusterNamespace, true, false)
```

Comment:
I'm curious why we are taking this approach rather than allowing users to use `ceph config set osd osd_mclock_profile <option>` directly. 

It seems like this feature is a one-to-one passthrough where the CLI is just renaming one config to another. The CLI is effectively acting as an allow-list for configurations that users can set.

Will we be playing catch-up to implement `odf-cli set <option>` for any Ceph tunable that users want to set if we only implement some of them?

Reply:
>I'm curious why we are taking this approach rather than allowing users to use ceph config set osd osd_mclock_profile <option> directly.

The request from the customer was they didn't want to run the command from the toolbox they wanted something from odf.

>Will we be playing catch-up to implement odf-cli set <option> for any Ceph tunable that users want to set if we only implement some of them?

I think the purpose of adding this command is we decided to not expose all the ceph config options to customers directly and restrict customers to what config they can set based on their requests.

Also, what I have heard is that the toolbox is not always available to customers.

Reply:
Thanks for answering this and my other questions. Those reasons all make sense.